### ISSUE TITLE
xml vector icons does not show in met_iconLeft/met_iconRight

### ISSUE THREAD

**Issue by bluevoidnl** [rengwuxian/MaterialEditText#322]
I created a material icon by exporting it as a vector .xml in Android studio. Sing the drawable as a xml will not show it. Solved it by downloading the icon from material.io as pngs, that does work.  

 <com.rengwuxian.materialedittext.MaterialEditText
                android:id="@+id/edit_field_password"
                style="@style/MaterialEditText"
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:imeOptions="actionNext"
                android:inputType="textPassword"
                app:met_floatingLabel="highlight"
                app:met_iconLeft="@drawable/ic_https_black_24dp"
                />

**Comment by pcegarra** [comment]
Yeah, could be great add svgs

**Comment by 7Bit** [comment]
This should definately be adressed, as using xml vector icons is the preferred way to use drawable in Android.

**Comment by SabNK** [comment]
Is there any progress?

**Comment by JonathanNet** [comment]
Yeah I have the same problem here, can't use vector icons only png, any progress on that yet?

**Comment by anas04-haoud** [comment]
any solution for this problem??
